README.md

```diff
@@ -1 +1,11 @@
+# Arcade-extensions
 
+This repo contains the custom build tasks used by dnceng which are distributed in the ["DncEng Build Tasks"](https://marketplace.visualstudio.com/items?itemName=dotnet-dnceng.dnceng-build-release-tasks&targetId=b55de4ed-4b5a-4215-a8e4-0a0a5f71e7d8&utm_source=vstsproduct&utm_medium=ExtHubManageList) extension.
+
+## Overview
+
+[Tasks overview](https://github.com/dotnet/arcade/blob/master/Documentation/Projects/DevOps/Tasks/OnePager.md)
```

Comment:
Should we move this one pager over this repo?

Reply:
@garath?

Reply:
I don't see any benefit in moving it, yet.